composer.json

```diff
@@ -49,8 +49,9 @@
         "monolog/monolog": "~1",
         "psr/http-message": "1.0.*",
         "ramsey/uuid": "~3",
-        "google/proto-client": "0.27.0",
-        "google/gax": "0.27.0"
+        "google/proto-client": "^0.29.0",
+        "google/gax": "^0.28.0",
+        "google/cloud-bigquerydatatransfer": "^0.1.0"
```

Comment:
`google/cloud-bigquerydatatransfer` needs to be published and tagged before merging this PR.

Reply:
Yep. Good call.